CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- **[UPDATE]** Add `title` and `tabIndex` props to `A11yProps` interface and `pickA11yProps` function
```

Comment:
It's more a breaking change, as some typings are being changed

Reply:
Yes it'll require some components prop change on Kairos side. So I pass those changes to [BREAKING CHANGES].